packages/luna/src/components/PlanCard/PlanCard.tsx

```diff
@@ -0,0 +1,206 @@
+        {discount ? (
+          <div className="tw-self-stretch">
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                "tw-text-brand-700 tw-py-[2px] tw-px-3 tw-rounded-[30px]",
+                primary ? "tw-bg-discount-primary" : "tw-bg-discount-default"
+              )}
+            >
+              {intl("plan.discount", { value: formatNumber(discount) })}
+            </Typography>
+          </div>
+        ) : null}
+      </div>
+      <Typography
+        element="tiny-text"
+        weight="semibold"
+        className={cn(
+          "tw-inline-block tw-mb-4",
+          primary ? "tw-text-natural-100" : "tw-text-natural-700"
+        )}
+      >
+        {description}
+      </Typography>
+      <Typography
+        element="h4"
+        weight="bold"
+        className={cn(
+          "tw-mb-2",
+          primary ? "tw-text-natural-50" : "tw-text-brand-700"
+        )}
+      >
+        {intl("plan.weekly-minutes", { value: formatNumber(weeklyMinutes) })}
+      </Typography>
+      <div className="tw-flex tw-gap-2 tw-items-end tw-flex-wrap">
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {formatNumber(priceAfterDiscount, {
+            maximumFractionDigits: 2,
+          })}
+        </Typography>
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {intl("labels.currency.egp")}
+        </Typography>
+        {discount ? (
+          <div className="tw-flex tw-gap-1">
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                primary ? "tw-text-natural-100" : "tw-text-natural-700"
+              )}
+            >
+              {intl("plan.instead-of")}
+            </Typography>
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                "tw-block tw-relative",
+                primary ? "tw-text-destructive-400" : "tw-text-destructive-600"
+              )}
+            >
+              {intl("plan.price-before-discount", {
+                value: formatNumber(price),
+              })}
+              <span
+                className={cn(
+                  "tw-absolute tw-block tw-top-1/2 tw-right-0 tw-left-0 tw-border-t",
+                  primary
+                    ? "tw-border-destructive-400"
+                    : "tw-border-destructive-600"
+                )}
+              />
+            </Typography>
+          </div>
+        ) : null}
+      </div>
+      {primary ? (
+        <button
+          type="button"
+          className={cn(
+            "tw-w-full tw-py-3 tw-rounded-lg tw-mt-6",
```

Comment:
Add transition on colors with 300ms duration.

Reply:
@neuodev  
the button in the design has 200ms duration 
do you think it will be better if the 2 values have the same duratiom wheater 200 or 300

Reply:
Let's stick to the design.